### PR TITLE
Loyalty points expiration model updates

### PR DESCRIPTION
**Pull request by JonasSteur**
https://vikingco.atlassian.net/browse/INT-4836

### CODE REVIEW

File: definitions/customers/loyalty_program_data.yaml [hunk 1]
```diff
@@ -0,0 +1,26 @@
+type: object
+title: Loyalty Program Data
+description: |
+  Information about the status of a customer's loyalty points
```
Comment (freeduerinckx) [inline definitions/customers/loyalty_program_data.yaml]: This is incorrect. Even if you don't have Loyalty points you still would have this state.

File: definitions/customers/loyalty_program_data.yaml [hunk 2]
```diff
@@ -0,0 +1,26 @@
+type: object
+title: Loyalty Program Data
```
Comment (freeduerinckx) [inline definitions/customers/loyalty_program_data.yaml]: We shouldn't expose the bad names from SBSS. I'll update if i find a better name but this one is not very descriptive.

**Comment by nielsvanoch** [comment]
Frontend-wise, I see no problems.

File: definitions/customers/loyalty_program_data.yaml (hunk 2 above)
Comment (Timvde) [reply definitions/customers/loyalty_program_data.yaml]: Is it necessary to expose the state? Would it be enough to only expose a `loyalty_points_expiration_date' that can be either 1) left out (`ENABLED`), 2) in the future (`CUSTOMER_ONLY`) or 3) in the past (`SLEEP`)?

Comment (JonasSteur) [reply definitions/customers/loyalty_program_data.yaml]: @freeduerinckx I kind of like Tim's idea here. What do you think ? Would remove the need of having a separate serializer etc for this.

Comment (freeduerinckx) [reply definitions/customers/loyalty_program_data.yaml]: Both 1 and 3 will be in the future if not mistaking. The message the front end can be different.
In the first case you want to encourage a customer to activate it's sim. In the 3rd case you want to encourage a topup.

**Comment by JonasSteur** [comment]
@freeduerinckx  pushed a fix using the `loyalty_state_validity` name

**Pull request merged**